x/slashing/keeper_contracts.go

```diff
@@ -0,0 +1,9 @@
+package slashing
```

Comment:
i'd prefer to keep the name `expected_keepers.go` - keeper_contracts isn't bad either - but I would prefer consistency with the other modules